source/vocab/enums.ttl

```diff
@@ -93,6 +94,17 @@
     rdfs:label "Frekvens"@sv ;
     owl:equivalentClass bf2:Frequency .
 
+:pubFrequency a owl:ObjectProperty;
+    # No relation in bf to frequency?
+    :category :pending ;
+    rdfs:range :PubFrequency ;
+    owl:equivalentProperty bf2:pubFrequency .
+
+:PubFrequency a owl:Class;
```

Comment:
Do we have some (marc) notion for this in the ... 008(?) mappings?

Reply:
Rationale for this addition is discussed here https://github.com/lcnetdev/bibframe-ontology/issues/76

We have an issue to add it properly later, this is just to hide it from the cataloging tool until then.